it/api/UnignoreStateBenefitITest.scala

```diff
@@ -0,0 +1,196 @@
+      authorised()
+
+      whenReady(buildClient(serviceUrl)
+        .withHttpHeaders(mtditidHeader)
+        .delete) {
+        result =>
+          result.status mustBe NO_CONTENT
+      }
+    }
+
+    "return a 403 if a downstream Forbidden error occurs" in new Setup {
+      val errorResponseBody: String = Json.toJson(DesErrorBodyModel(
+        "CUSTOMER_ADDED", "The remote endpoint has indicated you Cannot un-ignore customer added State Benefit."
+      )).toString()
+
+      val expectedResult: JsValue =  DesErrorModel(FORBIDDEN, DesErrorBodyModel.forbiddenUnignore).toJson
+
+      stubDeleteWithResponseBody(desUrl, FORBIDDEN, errorResponseBody)
+      authorised()
+
+      whenReady(buildClient(serviceUrl)
+        .withHttpHeaders(mtditidHeader)
+        .delete()) {
+        result =>
+          result.status mustBe FORBIDDEN
+          Json.parse(result.body) mustBe expectedResult
+      }
+    }
+
+    "return a 422 if a downstream Forbidden error occurs" in new Setup {
+      val errorResponseBody: String = Json.toJson(DesErrorBodyModel(
+        "BEFORE_TAX_YEAR_ENDED", "Submission is provided before the tax year has ended."
+      )).toString()
+
+      val expectedResult: JsValue =  DesErrorModel(FORBIDDEN, DesErrorBodyModel.beforeTaxYear).toJson
+
+      stubDeleteWithResponseBody(desUrl, UNPROCESSABLE_ENTITY, errorResponseBody)
+      authorised()
+
+      whenReady(buildClient(serviceUrl)
+        .withHttpHeaders(mtditidHeader)
+        .delete()) {
+        result =>
+          result.status mustBe UNPROCESSABLE_ENTITY
+          Json.parse(result.body) mustBe expectedResult
+      }
+    }
+
+    "return a 400 if a downstream Bad Request error occurs" in new Setup {
+      val errorResponseBody: String = Json.toJson(DesErrorBodyModel(
+        "INVALID_BENEFIT_ID", "Submission has not passed validation. Invalid parameter benefitId."
+      )).toString()
+
+      val expectedResult: JsValue =  DesErrorModel(BAD_REQUEST, DesErrorBodyModel.invalidBenefitId).toJson
+
+      stubDeleteWithResponseBody(desUrl, BAD_REQUEST, errorResponseBody)
+      authorised()
+
+      whenReady(buildClient(serviceUrl)
+        .withHttpHeaders(mtditidHeader)
+        .delete()) {
+        result =>
+          result.status mustBe BAD_REQUEST
+          Json.parse(result.body) mustBe expectedResult
+
+
+      }
+    }
+
+    "return a 404 if no data is found to delete" in new Setup {
+      val errorResponseBody: String = Json.toJson(DesErrorBodyModel(
+        "NO_DATA_FOUND", "The remote endpoint has indicated that the requested resource could not be found."
+      )).toString()
+
+      stubDeleteWithResponseBody(desUrl, NOT_FOUND, errorResponseBody)
+      authorised()
+
+      whenReady(buildClient(serviceUrl)
+        .withHttpHeaders(mtditidHeader)
+        .delete()) {
+        result =>
+          result.status mustBe NOT_FOUND
+          Json.parse(result.body) mustBe Json.obj(
```

Comment:
For this and the remaining tests, could probably make use of the existing DesErrorModels as you have done above. I know it wasn't really done in some earlier tickets we did but I think it makes sense to do 

e.g. DesErrorModel(NOT_FOUND, DesErrorBodyModel.noDataFound).toJson